src/main/resources/application-dev.properties

```
@@ -15,4 +15,13 @@ spring.jpa.properties.hibernate.show-sql=true
 logging.level.org.hibernate.type.descriptor.sql=DEBUG
 
 logging.level.org.hibernate.SQL=DEBUG
-spring.jpa.properties.hibernate.dialect= org.hibernate.dialect.MariaDBDialect
\ No newline at end of file
+spring.jpa.properties.hibernate.dialect= org.hibernate.dialect.MariaDBDialect
+
+### Redis
+#spring.data.redis.host=/TODO # development Redis Endpoint
+#spring.data.redis.port=/TODO # development Redis Port
+
+### JWT
+# TODO : need to fill secret
+spring.jwt.secret=
```

```suggestion
spring.jwt.secret=YWxsLXRpbWUtY2F0Y2hlci1qd3Qtc2VjcmV0LWNvbmZpZ3VyYXRpb24tZGV2bC1zb3VyY2UtY29kZQo=
```
이거 application.properties 파일에 있는 암호화값인데 위처럼 되어 있으면 빈값으로 override 할것 같으니 없애거나 다 채워주세요!